ee/tabby-db/src/threads.rs

```diff
@@ -220,6 +220,24 @@ impl DbConn {
         Ok(())
     }
 
+    pub async fn update_thread_message_content(
+        &self,
+        thread_id: i64,
+        message_id: i64,
```

Comment:
need to verify message_id belongs to thread_id

Reply:
https://github.com/TabbyML/tabby/pull/3344/files/640aeaad3e2819cbe28e6481d27b52859f65ae74#diff-d0fb876834081b31a6bf3d505dfb7c1567ed9fe5c851ac4fa0387f4437b41856R230 

use condition `thread_id = ? and id = ?`, it should report NOT_FOUND when the message_id not belongs to thread_id